spec/coffee_place/geo_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'coffee_place/geo'
+
+RSpec.describe CoffeePlace::Geo do
+  subject(:geo) { described_class }
+
+  it 'checks whether a value is within latitude bounds' do
+    expect(geo.within_latitude_bounds?(0)).to eq(true)
+    expect(geo.within_latitude_bounds?(42.0)).to eq(true)
+    expect(geo.within_latitude_bounds?(-42.0)).to eq(true)
+
+    expect(geo.within_latitude_bounds?(90.0)).to eq(true)
+    expect(geo.within_latitude_bounds?(-90.0)).to eq(true)
+
+    expect(geo.within_latitude_bounds?(92.0)).to eq(false)
+    expect(geo.within_latitude_bounds?(-92.0)).to eq(false)
```

Comment:
Hm, why did you choose to break the rule of having one expectation per test?

Reply:
Having all of these assertions in the same example here is **pure laziness (tm)** on my part.
I'll try to add a few more commits, describing the edge cases I am testing a bit better.

That being said, I **don't believe** in the one expectation per test rule.
I think a test should describe and test the expectations of a specific scenario.

I would rather have 7 expectations in the same example, testing a single well described scenario,
than break those 7 expectations into different examples, all handling a single path through the code.

Reply:
Update: fixed by [033654a3c23b2233b](https://github.com/Agilefreaks/test_oop/pull/23/commits/033654a3c23b2233b815e96230dc2d1cf49469fc)